src/controllers/auth.controllers.ts

```diff
@@ -0,0 +1,15 @@
+import { Request, Response } from "express";
+import { loginService } from "../services/auth.services";
+import { handleError } from "../utils";
+
+const login = async (req: Request, res: Response): Promise<void> => {
```

Comment:
I highly recommend to use try and catch on the controllers, because we are not using them on the services but we are using async functions and if we have some error to catch maybe it will crash the rest api.

Reply:
i just went through the code and i'm afraid that adding the exception handling to the code would be useless option
cause when i went through code inside service  there is no function that gonna raise an issue
- findOne actually return the current user if the user is found  and null if there is no user with the provided username
- bcrypt.match returns boolean